pkg/test/resourcefixture/testdata/basic/tags/v1beta1/tagstagbinding/tagbindingbasic/create.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
   name: tagstagbinding-${uniqueId}
 spec:
   parentRef:
-    name: project-${uniqueId}
+    # TODO: projectId should work
```

Comment:
should we add an issue to investigate here? I'm asking since we are modifying the dependencies